openc3-init/plugins/packages/openc3-tool-cmdsender/src/tools/CommandSender/CommandSender.vue

```diff
@@ -681,18 +589,20 @@ export default {
       this.displaySendHazardous = false
       var paramList = this.createParamList()
 
+      // Ignore 500 errors because we might get a 500 error on out of range which we handle
+      localStorage.axiosIgnoreResponse = '500' // localStorage only supports strings
```

Comment:
We have to have a better way of doing this.  This is a global that will prevent 500 errors from this point forward, if for some reason it doesn't get unset.

Reply:
Agreed. Created https://github.com/OpenC3/openc3/issues/107 to address.